Facepunch.Steamworks/SteamRemoteStorage.cs

```diff
@@ -79,6 +79,22 @@ public unsafe static byte[] FileRead( string filename )
 		public static bool FileDelete( string filename ) => Internal.FileDelete( filename );
 
 
+		/// <summary>
+		/// Marks the file as shared so other users can download it
+		/// </summary>
+		public static async Task<ulong> FileShare(string filename)
+		{
+			var result = await Internal.FileShare(filename);
+
+			if (!result.HasValue)
+				return ulong.MinValue;
```

Comment:
Would make more sense to return ulong? and null here

Reply:
I was just following what I saw other functions doing, for consistency

Reply:
What other function returns ulong.MinValue?

Reply:
it's been too long since i worked on this, i can't remember. I can change it to null if that makes more sense here!